test/EndToEnd/tests/ServicesTest.ps1

```diff
@@ -703,6 +703,40 @@ function Test-BatchEventsApi
     Assert-True $result
 }
 
+function Test-ExecuteInitScriptsPerSolution
```

Comment:
I know, but i cant find a way to restart the VS itself. Do you know how could we achieve it in our existing infrastructure?